aten/src/ATen/native/cuda/Reduce.cuh

```diff
@@ -344,13 +365,13 @@ struct ReduceOp {
     extern __shared__ char shared_memory[];
     index_t output_idx = config.output_idx<output_vec_size>();
     index_t input_idx = config.input_idx();
-    auto base_offsets1 = output_calc.get(output_idx)[1];
+    auto base_offsets_input = output_calc.get(output_idx)[num_outputs];
```

Comment:
`num_outputs` and `config.num_outputs` are totally different things, right? `num_outputs` is number of output tensors, usually 1, but 2 for min/max operations, and `config.num_outputs` is number of outputs produced by a single thread?

Reply:
Yes they are totally different things. And `config.num_outputs` I think it is `iter.num_output_elements()` which is the numel of output tensor.

Reply:
I renamed `config.num_outputs` to `config.num_output_elements`